lib/Github/Api/Repo.php

```diff
@@ -60,6 +60,23 @@ public function all($id = null)
     }
 
     /**
+       * List all repositories owned by the user.
+       *
+       * @link https://developer.github.com/v3/repos/#list-all-public-repositories
```

Comment:
This is wrong link. Should be: https://developer.github.com/v3/repos/#list-your-repositories